.github/workflows/build.yml

```
@@ -196,7 +196,7 @@ jobs:
           # This is a workaround for a SSL false positive in cmake 3.26.4
           # When downloading the manual. 3.21 is required for installing the
           # ANGLE Dlls via IMPORTED_RUNTIME_ARTIFACTS
```

The comment is from here: 
https://github.com/mixxxdj/mixxx/pull/12197
Can you verify that downloading the manual works with CMake 3.30 

```suggestion
          # On Windows, we need at least CMake 3.30.x for WIX toolset >= 4.0 
```